packages/flutter_blue/tizen/src/bluetooth_device_controller.h

```diff
@@ -0,0 +1,103 @@
+#ifndef FLUTTER_BLUE_TIZEN_BLUETOOTH_DEVICE_CONTROLLER_H
+#define FLUTTER_BLUE_TIZEN_BLUETOOTH_DEVICE_CONTROLLER_H
+
+#include "GATT/bluetooth_service.h"
+#include "bluetooth.h"
+#include "flutterblue.pb.h"
+#include "notifications_handler.h"
+#include "utils.h"
+
+namespace flutter_blue_tizen {
+
+class BluetoothDeviceController {
+ public:
+  enum class State {
+    kConnected,
+    kConnecting,
+    kDisconnected,
+    kDisconnecting,
+  };
+
+  using requestMtuCallback =
+      std::function<void(bool, const BluetoothDeviceController&)>;
+
+  BluetoothDeviceController(
+      const std::string& address,
+      NotificationsHandler& notifications_handler) noexcept;
+
+  BluetoothDeviceController(
+      const char* address,
+      NotificationsHandler& notifications_handler) noexcept;
+
+  ~BluetoothDeviceController() noexcept;
+
+  BluetoothDeviceController() = delete;
```

Comment:
This is not needed, default constructor will not be created when there's a custom constructor.